### ISSUE TITLE
Consider using purescript-cip30

### ISSUE THREAD

**Issue by klntsky** [Plutonomicon/cardano-transaction-lib#1115]
https://github.com/mlabs-haskell/purescript-cip30

Blocked by: #400

**Comment by uhbif19** [comment]
@klntsky Looks like there is little code and it is already covered. There was a pagination difference - but it will be covered by my PR.

We do not have these functions, but I am not sure if we need them:

```
foreign import _getName :: WalletName -> Effect String
foreign import _getIcon :: WalletName -> Effect DataUri
foreign import allWalletTags :: Effect (Array String)
```

**Comment by klntsky** [comment]
@uhbif19 we don't need allWalletTags, but we do need Name and Icon. Strange that we missed them. Please make an issue to add them, and thanks for noticing.

**Comment by jy14898** [comment]
I believe we do have name and icon, check `examples/Cip30.purs` @klntsky @uhbif19 

https://github.com/Plutonomicon/cardano-transaction-lib/blob/99acc5dacddf2b05be1dfcb9fd83b58671ac734c/examples/Cip30.purs#L44-L51
